operator-wg/whitepaper/03_Capabilities.md

```diff
@@ -20,20 +20,18 @@ An operator should monitor the update, and rollback if there was a problem durin
 An operator should report the version of the resources and their health status during the process. If there was an error, the version reported should be the version that is actually been used.
 
 #### Backup
-**(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/49**
 
-An operator’s capability to back up the application state and it’s data should give the user the certainty that he is able to restore from a failure if data is lost or compromised.
+This capability ensures that an operator is able to create consistent backups. This backup should be done in a way that the user of the operator can be certain that it can be restored if data is lost or compromised. Furthermore, the status information provided, should give insights when the backup ran the last time and where it is located.
 
-Therefore, the process of backing up data can be described as follows:
-Backup gets triggered (either manually or automatically)
-The application is set to a state where it allows consistent backups
-Backup data and save it to an external storage
-Write the Backup state and its location to the custom resource
+![Example Backup Process](plantuml/backup-sequence.png)
```

Comment:
Maybe add in the illustration the external storage, for clarity

Reply:
As there is a lot of ways to back up the data, i think the current drawing is the most neutral one (but i get your point) ...